page/ts/faqScroll.js

```diff
@@ -0,0 +1,73 @@
+/**
+ *
+ *  jQuery plugin that handling the smooth 
+ *  scrolling for FAQ page
+ */
+
+;(function ($) {
```

Comment:
Do you really need that funny pattern with injecting IIFE function? `ProvidePlugin` should inject `$`automatically.  so you dont need to wrap your code in that function